src/Components/Artwork/Details.tsx

```diff
@@ -246,6 +250,15 @@ export const Details: React.FC<DetailsProps> = ({
       return renderSaveButton(rest.artwork.internalID)
     }
 
+    if (isArtworksListEnabled && enableSaveButtonForLists) {
+      return (
+        <SaveArtworkToListsButtonFragmentContainer
```

Comment:
Will this eventually replace the other button on L263?

Reply:
Yes, that's right